tensorboard/components/tf_data_selector/tf-data-selector.ts

```diff
@@ -33,13 +33,17 @@ Polymer({
       computed: '_getComparingExps(_comparingExpsString, _allExperiments.*)',
     },
 
+    // TODO(stephanwlee): Add list of active plugin from parent and filter out
```

Comment:
Hmm, I don't see logic here that populates the .tags field on the runs in _selectionMap?  But categorizeSelection() expects the field to be there on the selection it receives, which is generated from _selectionMap (via this._setSelection(), which as an aside doesn't actually seem to be defined anywhere?).

I think the fireChange() code in tf-data-select-row would need to populate that field when propagating the run information up from the individual row to the overall data selector.

Reply:
tags field has been populated from BE but FE omitted it from the type declaration.

Like you indicated, it is already populated in the tf-data-select-row. This comment is about maybe filtering out tags for inactive plugins in the tf-data-select-row.

Reply:
I understand the TODO comment is not exactly related to my comment, but GitHub only lets you comment on changed lines +/- 3 lines so I just picked a line that seemed roughly relevant.

How does the tags field get propagated into _selectionMap?  The _selectionChanged() handler here passes through event.detail.runs, and in tf-data-select-row the _fireChange() method that fires `selection-changed` does so by passing `runs` as defined to contain only the id, name, and startTime fields:
https://github.com/stephanwlee/tensorboard/blob/84026254338fb3436339270d10f478d25b0f3fbd/tensorboard/components/tf_data_selector/tf-data-select-row.ts#L187-L193

So even if tf-data-select-row._runs contains the .tags field fetched from the backend, it looks to me like it would be dropped in _fireChanged().

Reply:
It seems like I have sycned my local to a wrong version and was not up-to-date.
Addressed. Thanks for catching that!

Reply:
Looks good, thanks!